openlane/scripts/openroad/common/io.tcl

```diff
@@ -415,7 +429,20 @@ proc max {a b} {
 
 set ::metric_count 0
 set ::metrics_file ""
-if { [info exists ::env(_OPENSTA)] && $::env(_OPENSTA) } {
+if { [namespace exists utl] } {
```

Comment:
is this namespace specific to opensta?

Reply:
inverse, it's OpenROAD-specific, and in this exact scenario it doesn't matter because if `utl` exists I want to use it